CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ In [4]: from pyqasm import dumps
 In [5]: dumps(module).splitlines()
 Out[5]: ['OPENQASM 3.0;', 'qubit[2] q;', 'h q;']
 ```
+- Added support for unrolling multi-bit branching with `==`, `>=`, `<=`, `>`, and `<`.
```

Comment:
Can you add an example here showing the qasm before and after unrolling a program?